### PR TITLE
week 4 homework

### PR DESCRIPTION
**Pull request by estr**
Homework week 4

### CODE REVIEW

File: starter_code/js/app.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	});
+
+	$(".readmore").click(function(){
+		$("#show-this-on-click").slideDown();
+		$(".readless").show();
+		$(".readmore").hide();		
+	});
+
+	$(".readless").click(function() {
+		$("#show-this-on-click").slideUp();
```
Comment (yangli-io) [inline starter_code/js/app.js:13]: You can add a function in slideUp/SlideDown for actions you want to execute when the sliding action has finished

```
$('#show-this-on-click').slideUp(function(){
    $('.readless').hide();
    $('.readmore').show();
});
```

For a better UX

File: starter_code/js/app.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function main() {
+	$( "a" ).click(function(event) {
+  		event.preventDefault()
```
Comment (yangli-io) [inline starter_code/js/app.js:3]: Great! But becareful, if you want to add a real link to the page - it won't work and you might spend hours scratching your head why it's not working.

**Comment by yangli-io** [comment]
Great stuff

File: starter_code/js/app.js (hunk 1 above)
Comment (estr) [inline starter_code/js/app.js:13]: Yes!! That's what I was trying to do and couldn't work out, much nicer!

**Pull request closed without merging**